runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -0,0 +1,411 @@
+                            log.debug("[{}] {}", clientName, "Not all destinations to remove are present.");
+                        }
+
+                        if (clientDestinationIdsList.size() == 0) {
+                            txn.delete(sourceMetadataTable, clientInfoKey);
+                        } else {
+                            clientDestinationIds = clientDestinationIds.toBuilder()
+                                    .clearDestinationIds()
+                                    .addAllDestinationIds(clientDestinationIdsList)
+                                    .build();
+                            txn.putRecord(sourceMetadataTable, clientInfoKey, clientDestinationIds, null);
+                        }
+                        txn.commit();
+
+                        return null;
+                    }
+                    throw new NoSuchElementException(String.format("Record not found for group: %s", logicalGroup));
+                } catch (TransactionAbortedException tae) {
+                    log.error("[{}] {}: {}", clientName, "Unable to remove destinations, will be retried", tae);
+                    throw new RetryNeededException();
+                }
+            }).run();
+        } catch (InterruptedException e) {
+            log.error("[{}] {}: {}", clientName, "Unable to remove destinations", e);
+            throw new UnrecoverableCorfuInterruptedError(e);
+        }
+    }
+
+    /**
+     * Print the current destinations for a logical group
+     *
+     * @param logicalGroup Group which will have its destinations printed
+     */
+    public void showDestinations(String logicalGroup) {
+        Preconditions.checkArgument(isValid(logicalGroup), String.format("[%s] %s", clientName, "logicalGroup is null or empty."));
+        try {
+            IRetry.build(ExponentialBackoffRetry.class, () -> {
+                try (TxnContext txn = corfuStore.txn(CORFU_SYSTEM_NAMESPACE)) {
+                    ClientDestinationInfoKey clientInfoKey = ClientDestinationInfoKey.newBuilder()
+                            .setClientName(clientName)
+                            .setModel(LogReplication.ReplicationModel.valueOf(modelEnum))
+                            .setGroupName(logicalGroup)
+                            .build();
+
+                    if (txn.isExists(LR_MODEL_METADATA_TABLE_NAME, clientInfoKey)) {
+                        DestinationInfoVal clientDestinationIds = txn.getRecord(sourceMetadataTable, clientInfoKey).getPayload();
+                        List<String> clientDestinationIdsList = new ArrayList<>(clientDestinationIds.getDestinationIdsList());
+
+                        System.out.printf("\n========== Destinations for %s ==========\n", logicalGroup);
+                        for (String destination : clientDestinationIdsList) {
+                            System.out.println(destination);
+                        }
+                        System.out.println("==================================\n");
```

Comment:
Please use Slf4j logger instead. Also, did you run the added tests locally? the violation of Checkstyle fails the build consistently on my end

Reply:
I did but changed to print statements after build to test something and forget to revert before committing